dashboard/app/views/layouts/application.html.haml

```diff
@@ -10,6 +10,7 @@
       %meta{'http-equiv' => 'X-UA-Compatible', content: 'IE=Edge'}
     = stylesheet_link_tag    'application', media: 'all'
     = javascript_include_tag 'application'
+    = javascript_include_tag "js/#{js_locale}/common_locale"
```

Comment:
The change was because we have apps strings we need in code-studio, i.e. wherever code-studio-common is included. This was resulting in a double inclusion, which I fixed by removing from _apps_dependencies, and then adding to embed.